### PR TITLE
Added config option for overlay

### PR DESCRIPTION
**Pull request by CoryRobertson**
Added config option to overlay so you can just have the gold drops
By the way, this is my first time working with runelite plugins, I'd love to get some feedback if this is a good change at all, or if there are better ways this can be done! Thanks for your time!

### CODE REVIEW

File: src/main/java/com/profittracker/ProfitTrackerOverlay.java [hunk 1]
```diff
@@ -45,63 +46,61 @@ public Dimension render(Graphics2D graphics) {
         String titleText = "Profit Tracker:";
         long secondsElapsed;
         long profitRateValue;
+        if(ptConfig.ingameOverlay()) {
```
Comment (nofatigue) [inline src/main/java/com/profittracker/ProfitTrackerOverlay.java:49]: Hey, I see this "IF" is the main change in the pull request. I think a nicer way to do this is
reversing the if:
`if (!ptConfig.ingameOverlay()) {
    return null;
}
`
Thus avoiding all the extra indentation.

Comment (CoryRobertson) [reply src/main/java/com/profittracker/ProfitTrackerOverlay.java:49]: Oh you are totally right, I don't see why I didn't do that first, good idea. Well thanks for looking at my pull anyway (: it's my first time

**Comment by nofatigue** [comment]
Hi, 
I'm glad you wanted to work with this code and add a feature :)
I've added a comment on your code.
But I must inform you:  Since I own the repo but no longer actually maintain it, even If the pull requested is merged, I might not actually update it with the runelite plugin repo. And another thing: Currently I'm talking with someone that might fork and update the repo in the future.  So in case he forks before your code is merged, your code will not be in his fork. I'm sorry for this timing. If you choose to close this pull request or leave it as it is -  it is understandable.

**Comment by CoryRobertson** [comment]
> Hi,
> I'm glad you wanted to work with this code and add a feature :)
> I've added a comment on your code.
> But I must inform you: Since I own the repo but no longer actually maintain it, even If the pull requested is merged, I might not actually update it with the runelite plugin repo. And another thing: Currently I'm talking with someone that might fork and update the repo in the future. So in case he forks before your code is merged, your code will not be in his fork. I'm sorry for this timing. If you choose to close this pull request or leave it as it is - it is understandable.

That's totally ok, I more just wanted to learn how making a runelite plugin works and wanted to get the experience of making a pull request, no worries at all!

**Pull request closed without merging**